src/styles/styles.js

```diff
@@ -2858,12 +2849,6 @@ const styles = {
         paddingRight: 18,
     },
 
-    pushToPageEmptyItemLabel: {
-        color: themeColors.textSupporting,
-        fontSize: variables.fontSizeNormal,
-        maxWidth: 240,
```

Comment:
We had this maxWidth in other components, but it actually caused [this bug](https://github.com/Expensify/App/pull/14040) and the solution was to remove it. So I'm removing it here too!